js/index.js

```diff
@@ -1,9 +1,14 @@
 import { SlideDeck } from './slidedeck.js';
 
-const map = L.map('map', {scrollWheelZoom: false}).setView([0, 0], 0);
+const mapElement = document.querySelector('#map');
+const map = L.map(mapElement).setView([35.652, 139.839], 5);
```

Comment:
**issue (map zoom):** The map object need to have `{scrollWheelZoom: false}` to prevent zooming in and out the map as people are scrolling.      
**suggestion:** The `L.map` function can do what you tried to do in the above line as `L.map('map')`, which can query the `#map` tag in your HTML.    

```suggestion
const map = L.map('map', {scrollWheelZoom: false}).setView([35.652, 139.839], 5);
```